Marlin/src/lcd/language/language_it.h

```diff
@@ -180,6 +180,7 @@
 #define MSG_MOVE_Y                          _UxGT("Muovi Y")
 #define MSG_MOVE_Z                          _UxGT("Muovi Z")
 #define MSG_MOVE_E                          _UxGT("Estrusore")
+#define MSG_HOTEND_TOO_COLD                 _UxGT("Ugello troppo freddo")
```

Comment:
This is one letter too long to fit on the HD44780 screen.

Reply:
How about just "L'ugello è freddo"

Reply:
In English this will sound like "The hotend is cold" but that's also good.
We may think also at "Ugello freddo" ("Cold hotend")
I'll fix it this evening